src/app/App.js

```diff
@@ -91,7 +91,7 @@ class Main extends React.Component {
 
   render() {
     return (
-      <div style={{paddingTop: this.props.location.query.hideNavbar ? '0px' : '90px'}}>
+      <div style={{paddingTop: this.props.location.query.hideNavbar ? '0px' : '0px'}}>
```

Comment:
Shouldn't this just be paddingTop: '0px' if that's what it's being set to no matter what?